app/uk/gov/hmrc/calculatepublicpensionadjustment/controllers/SubmissionController.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.calculatepublicpensionadjustment.controllers
+
+import cats.data.EitherT
+import play.api.Logging
+import play.api.libs.json.{JsSuccess, JsValue, Json, Reads}
+import play.api.mvc._
+import uk.gov.hmrc.calculatepublicpensionadjustment.models.submission.{RetrieveSubmissionResponse, Submission, SubmissionRequest, SubmissionResponse}
+import uk.gov.hmrc.calculatepublicpensionadjustment.services.SubmissionService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendBaseController
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SubmissionController @Inject() (
+  override val controllerComponents: ControllerComponents,
+  submissionService: SubmissionService
+)(implicit ec: ExecutionContext)
+    extends BackendBaseController
+    with Logging {
+
+  def submit: Action[JsValue] = Action.async(parse.json) { implicit request =>
```

Comment:
Again it might be intentional but this method isnt used yet

Reply:
It services the POST /submission endpoint as per app.routes, and is tested in SubmissionControllerSpec.